### PR TITLE
fix: MIN_NORMAL_VALUE and add POSITIVE_INFINITY/NEGATIVE_INFINITY

### PR DESCRIPTION
**Pull request by MaxGraey**
+ fix MIN_POSITIVE_VALUE -> MIN_NORMAL_VALUE in definitions
+ add (f32|f64).POSITIVE_INFINITY and (f32|f64).NEGATIVE_INFINITY
+ add (f32|f64).NaN fo consistany
- [x] I've read the contributing guidelines

### CODE REVIEW

File: std/portable/index.js [hunk 1]
```diff
@@ -80,7 +80,9 @@ Object.defineProperties(
     "MAX_VALUE": { value: Math.fround(3.4028235e+38), writable: false },
     "MIN_NORMAL_VALUE":  { value:  Math.fround(1.17549435e-38), writable: false },
     "MIN_SAFE_INTEGER":  { value: -16777215, writable: false },
-    "MAX_SAFE_INTEGER":  { value:  16777215, writable: false }
+    "MAX_SAFE_INTEGER":  { value:  16777215, writable: false },
+    "POSITIVE_INFINITY": { value:  Math.fround(Infinity), writable: false },
+    "NEGATIVE_INFINITY": { value: Math.fround(-Infinity), writable: false }
```
Comment (dcodeIO) [inline std/portable/index.js:85]: The `Math.fround(+/-Infinity)` here looks like it has no effect in JS due to the final value effectively being a `number` anyway.

Comment (dcodeIO) [reply std/portable/index.js:85]: Makes me wonder about the ones above, but these may actually round to something different than the input? Or can we precompute these somehow?

Comment (MaxGraey) [reply std/portable/index.js]: Makes sense. I precomputed all this values

Comment (dcodeIO) [reply std/portable/index.js]:
```
> Math.fround(1.1920929e-07) == 1.1920928955078125e-07
true
> Math.fround(1.4012985e-45) == 1.4012984643248170e-45
true
> Math.fround(3.4028235e+38) == 3.4028234663852886e+38
true
> Math.fround(1.17549435e-38) == 1.1754943508222875e-38
true
> Math.fround(Infinity) == Infinity
true
> Math.fround(-Infinity) == -Infinity
true
```
👍

File: std/portable/index.d.ts [hunk 2]
```diff
@@ -230,11 +230,15 @@ declare namespace f32 {
...
-  export const MIN_POSITIVE_VALUE: f32;
+  export const MIN_NORMAL_VALUE: f32;
   /** Smallest safely representable integer value. */
   export const MIN_SAFE_INTEGER: f32;
   /** Largest safely representable integer value. */
   export const MAX_SAFE_INTEGER: f32;
+  /** Positive infinity value. */
+  export const POSITIVE_INFINITY: f32;
+  /** Negative infinity value. */
+  export const NEGATIVE_INFINITY: f32;
```
Comment (dcodeIO) [inline std/portable/index.d.ts:241]: So, previously these were present on `F32` analogous to JS's `Number` (but not on `f32`), with the other members of `F32` being defined as

```ts
class F32 {
  ...
  static readonly MAX_SAFE_INTEGER: f32 = f32.MAX_SAFE_INTEGER;
  ...
}
```

so perhaps it makes sense to also update

```ts
class F32 {
  ...
  static readonly POSITIVE_INFINITY: f32 = f32.POSITIVE_INFINITY;
  ...
  static readonly NEGATIVE_INFINITY: f32 = f32.NEGATIVE_INFINITY;
  ...
}
```

for consistency?

Comment (MaxGraey) [reply std/portable/index.d.ts:241]: Yeah. Also added f32.NaN and f64.NaN for consistency

File: std/assembly/builtins.ts [hunk 3]
```diff
@@ -925,6 +937,18 @@ export namespace f64 {
...
+  @lazy
+  export const POSITIVE_INFINITY: f64 =  Infinity;
+
+  // @ts-ignore: decorator
+  @lazy
+  export const NEGATIVE_INFINITY: f64 = -Infinity;
+
+  // @ts-ignore: decorator
+  @inline @lazy
+  export const NaN: f64 = 0.0 / 0.0;
```
Comment (MaxGraey) [inline std/assembly/builtins.ts:950]: can't use `export const NaN: f64 = NaN` here, so I decide use this workaround

Comment (dcodeIO) [reply std/assembly/builtins.ts:950]: Another workaround might be to import `NaN` from the same file under a different name, like `import { NaN as builtin_NaN } from "./number";` and then do `export const NaN: f64 = builtin_NaN;`. Not sure what's less strange.

Comment (MaxGraey) [reply std/assembly/builtins.ts:950]: I guess `import { NaN as builtin_NaN } from ..` more strange and expensive in term of compilation cost

File: std/assembly/builtins.ts [hunk 4]
```diff
@@ -925,6 +937,18 @@ export namespace f64 {
...
+  // @ts-ignore: decorator
+  @lazy
+  export const POSITIVE_INFINITY: f64 =  Infinity;
+
+  // @ts-ignore: decorator
+  @lazy
+  export const NEGATIVE_INFINITY: f64 = -Infinity;
+
+  // @ts-ignore: decorator
+  @inline @lazy
```
Comment (dcodeIO) [inline std/assembly/builtins.ts:949]:
```suggestion
  @lazy
```

File: std/assembly/builtins.ts [hunk 5]
```diff
@@ -846,6 +846,18 @@ export namespace f32 {
...
+  // @ts-ignore: decorator
+  @lazy
+  export const POSITIVE_INFINITY: f32 =  Infinity;
+
+  // @ts-ignore: decorator
+  @lazy
+  export const NEGATIVE_INFINITY: f32 = -Infinity;
+
+  // @ts-ignore: decorator
+  @inline @lazy
```
Comment (dcodeIO) [inline std/assembly/builtins.ts:858]:
```suggestion
  @lazy
```

Comment (dcodeIO) [reply std/assembly/builtins.ts]: (just so all of them are the same)

**Review by dcodeIO — APPROVED** [review]

**Comment by dcodeIO** [comment]
LGTM! We'll also have to update the docs with the new ones :)

**Comment by dcodeIO** [comment]
Looks like fixtures need to be updated for the suggestions

**Pull request merged**